package.json

```diff
@@ -48,28 +48,29 @@
     "lodash": "^4.17.11",
     "memdown": "^3.0.0",
     "multihashes": "~0.4.14",
-    "multihashing-async": "~0.5.2",
+    "multihashing-async": "multiformats/js-multihashing-async#feat/async-iterators",
     "ncp": "^2.0.0",
     "rimraf": "^2.6.3"
   },
   "dependencies": {
-    "async": "^2.6.2",
     "base32.js": "~0.1.0",
     "bignumber.js": "^8.1.1",
     "buffer": "^5.2.1",
     "cids": "~0.5.8",
-    "datastore-core": "~0.6.0",
-    "datastore-fs": "~0.8.0",
-    "datastore-level": "~0.10.0",
+    "datastore-core": "git://github.com/zcstarr/js-datastore-core.git",
+    "datastore-fs": "git://github.com/zcstarr/js-datastore-fs.git",
+    "datastore-level": "git://github.com/ipfs/js-datastore-level.git#refactor/async-iterators",
     "debug": "^4.1.0",
-    "dlv": "^1.1.2",
-    "interface-datastore": "~0.6.0",
+    "err-code": "^1.1.2",
+    "interface-datastore": "git://github.com/ipfs/interface-datastore.git#refactor/async-iterators",
```

Comment:
As the `refactor/async-iterators` PR was merged and the branch deleted this is outdated and breaks things.